common/main/es_AR.xml

```diff
@@ -149,14 +149,23 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 							<dayPeriod type="evening1" draft="contributed">tarde</dayPeriod>
 							<dayPeriod type="night1" draft="contributed">noche</dayPeriod>
 						</dayPeriodWidth>
+						<dayPeriodWidth type="narrow">
+							<dayPeriod type="am">a. m.</dayPeriod>
+							<dayPeriod type="noon">mediodía</dayPeriod>
+							<dayPeriod type="pm">p. m.</dayPeriod>
+							<dayPeriod type="morning1">madrugada</dayPeriod>
+							<dayPeriod type="morning2">mañana</dayPeriod>
+							<dayPeriod type="evening1">tarde</dayPeriod>
+							<dayPeriod type="night1">noche</dayPeriod>
+						</dayPeriodWidth>
 						<dayPeriodWidth type="wide">
 							<dayPeriod type="am" draft="contributed">a. m.</dayPeriod>
-							<dayPeriod type="noon" draft="contributed">↑↑↑</dayPeriod>
+							<dayPeriod type="noon" draft="contributed">mediodía</dayPeriod>
```

Comment:
There were no votes for this path in es_AR in v41, v42, v43, or v44.

The published value was "mediodía" for v41, v42, and v43. In other words, this change restores the specific value that was in the previous versions.

"wide" is inherited laterally from "abbreviated"

the parent locale es has a different value for this path: "del mediodía"